design.md

```diff
@@ -23,3 +23,15 @@
   
 ### login page
 ![main](imgs/login.png "login")
+
+
+### new event form
+![main](imgs/new_event.png "login")
+[source](https://wireframe.cc/nOFTKC)
+
+
+* The where will be a populated drop down menu (they didn't have a good one on the wireframe site)
+* group will be maintained in the database? 
+  * alternately should we create a "profile" page to set user details and maintain them?
```

Comment:
For now, just the database, to keep things simple?

Reply:
makes sense